`FixedDecimalFormatterOptions` does not implement the `Copy` trait

```
error[E0382]: borrow of moved value: `options`
   --> src/numberfmt.rs:185:56
    |
105 |     let mut options: options::FixedDecimalFormatterOptions = Default::default();
    |         ----------- move occurs because `options` has type `FixedDecimalFormatterOptions`, which does not implement the `Copy` trait
...
158 |         let fdf = FixedDecimalFormatter::try_new_unstable(&provider, &data_locale, options)
    |                                                                                    ------- value moved here
...
185 |         "actual_options": format!("{option_struct:?}, {options:?}"),
    |                                                        ^^^^^^^ value borrowed here after move
    |
    = note: this error originates in the macro `$crate::__export::format_args` (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/components/casemap/src/internals.rs
+++ b/components/casemap/src/internals.rs
@@ -23,7 +23,7 @@ const ACUTE: char = '\u{301}';
 
 // Used to control the behavior of CaseMapper::fold.
 // Currently only used to decide whether to use Turkic (T) mappings for dotted/dotless i.
-#[derive(Default)]
+#[derive(Copy, Clone, Default)]
 pub struct FoldOptions {
     exclude_special_i: bool,
 }

--- a/components/experimental/src/dimension/currency/options.rs
+++ b/components/experimental/src/dimension/currency/options.rs
@@ -6,7 +6,7 @@
 
 /// A collection of configuration options that determine the formatting behavior of
 /// [`CurrencyFormatter`](crate::dimension::currency::formatter::CurrencyFormatter).
-#[derive(Debug, Eq, PartialEq, Clone, Default)]
+#[derive(Copy, Debug, Eq, PartialEq, Clone, Default)]
 #[non_exhaustive]
 pub struct CurrencyFormatterOptions {
     /// The width of the currency format.

--- a/components/experimental/src/displaynames/options.rs
+++ b/components/experimental/src/displaynames/options.rs
@@ -21,7 +21,7 @@
 /// // Full name would be "Bosnia & Herzegovina"
 /// assert_eq!(display_name.of(region!("BA")), Some("Bosnia"));
 /// ```
-#[derive(Debug, Eq, PartialEq, Clone, Default)]
+#[derive(Copy, Debug, Eq, PartialEq, Clone, Default)]
 #[non_exhaustive]
 pub struct DisplayNamesOptions {
     /// The optional formatting style to use for display name.

--- a/components/segmenter/src/line.rs
+++ b/components/segmenter/src/line.rs
@@ -170,7 +170,7 @@ pub enum LineBreakWordOption {
 
 /// Options to tailor line-breaking behavior.
 #[non_exhaustive]
-#[derive(Clone, PartialEq, Eq, Debug)]
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
 pub struct LineBreakOptions {
     /// Strictness of line-breaking rules. See [`LineBreakStrictness`].
     pub strictness: LineBreakStrictness,

--- a/documents/process/graduation.md
+++ b/documents/process/graduation.md
@@ -27,6 +27,8 @@ This document contains a checklist for the requirements to migrate a component f
   - [ ] All options and conditional code paths should have a corresponding docs test with the heading `# Examples`
   - [ ] All functions that are conditional on a Cargo feature should say so (last line before `# Examples`): ```✨ *Enabled with the `alloc` Cargo feature.*```
   - [ ] Compiled data constructors should say "with compiled data" in the first sentence and should have a Cargo feature alert following the above syntax.
+- [ ] The APIs should follow ICU4X style
+  - [ ] All options bags should be `Copy` (and contain references if they need to). Exceptions can be made by discussion.
 - [ ] The data structs should fully follow ZeroVec style
   - [ ] Deserialization should not have a "zero-copy violation" in the [make-testdata](https://github.com/unicode-org/icu4x/blob/main/provider/datagen/tests/make-testdata.rs) test
   - [ ] Constructors should avoid allocating memory in the common case

